src/org/traccar/protocol/WatchFrameDecoder.java

```diff
@@ -33,7 +33,8 @@ protected Object decode(
             while (buf.readerIndex() < endIndex) {
                 byte b = buf.readByte();
                 if (b == '}') {
-                    switch (buf.readByte()) {
+                    byte c = buf.readByte();
```

Comment:
We need better names.

Reply:
like??? d instead of c? :) If b is good enough, then I can change c b2 (it's the byte after b) or ?

Reply:
B is short for Byte. C is nothing. Just a random letter. B2 is fine, but then rename first one to B1.